### DISCUSSION TITLE
How do filters work?

### DISCUSSION THREAD

**Discussion by Donkey-Doug** [Docile-Alligator/Infinity-For-Reddit#856] [Q&A] [answered]
So I press on the floating button and there is an animation, but nothing is filtered. I then press again on the floating button and now I can select some things to filter, but for some reason I can name and save my filter from this screen. So instead of having to manually select filters I tried to create default filters (like only gifs and videos), but I don't know how to apply a filter I created.

Also when not saving but just filtering, the results are very incomplete. For example, open any subreddit, filter on a type (say pics or text) and apply the filter. This then does not show all posts matching the filter description. I think filters in general are broken in this app.

And why can I not sort and filter at the same time?

**Comment by Wladefant** [comment, marked as answer]
https://github.com/Docile-Alligator/Infinity-For-Reddit/issues/1059
